podcasts/End of Year/Stories/IntroStory.swift

```diff
@@ -1,6 +1,6 @@
 import SwiftUI
 
-struct IntroStory: StoryView {
+struct IntroStory: ShareableStory {
```

Comment:
I remember that some people share a screenshot of the first and last story so I decided to just handle them. 🤷